### ISSUE TITLE
[Feature request]: Improvement to custody automatic logging on asset page

### ISSUE THREAD

**Issue by carlosvirreira** [Shelf-nu/shelf.nu#750]
### Is your feature request related to a problem? Please describe?

I have an admin and a user that have similar names. I want to be able to see the role of the user on the automatically logged mutation change. 

### Describe the solution you'd like

Can we show the user type somehow whenever a name is mentioned?

<img width="790" alt="Screenshot 2024-02-16 at 17 50 45" src="https://github.com/Shelf-nu/shelf.nu/assets/12449255/ab84a717-0fc2-4451-899d-ce323ed1c227">

**Comment by carlosvirreira** [comment]
Thoughts @jurrejansen @DonKoko

**Comment by DonKoko** [comment]
Yes we can. Do we do it just for the person who gave the custody?

**Comment by jurrejansen** [comment]
Yeah maybe as a quick fix put (admin) behind the name is user is admin or account owner. Can imagine some workspaces mostly have admins which will not look very nice / cluttered.

More long term solution would be the new design in which we can use badge with avatars and perhaps even a little icon on the avatar indicating admins.

![Screenshot 2024-02-19 at 11 19 26](https://github.com/Shelf-nu/shelf.nu/assets/76112272/4655c588-47d0-473a-bf01-499daf3e40dd)

But I agree with the (admin) behind name as a quick way to address this request

**Comment by DonKoko** [comment]
Yeah I think both those are kinda temp solutions. We would need to create something more powerful that solves this and many other issues related to identities. Like we also had a request from a user to actually show the email of the person. 
I think the best solution that is encompassing everything and will make it handle all cases is to create a popover badge for team members. So if you hover a team member anywhere(with some small exceptions) inside shelf you can see some relevant data like their role, email etc.

**Comment by jurrejansen** [comment]
A little preview of important team member info when hovering would indeed be nice and perhaps more realistic to build in the short term. Not ideal for mobile though.

Eventually we should work towards a dedicated team member page that can (also) be visited by clicking a team members badge.

![Screenshot 2024-03-27 at 11 37 30](https://github.com/Shelf-nu/shelf.nu/assets/76112272/a1d7bc57-c31e-405c-99ce-777f3d3b8dab)

**Comment by DonKoko** [comment]
Popovers work on mobile as well, just on tap rather than hover. Here is what we use, maybe you can give it a test: https://www.radix-ui.com/primitives/docs/components/popover

**Comment by DonKoko** [comment]
Keep in mind shadcn also has this specific component. I imagine it something like this: https://ui.shadcn.com/docs/components/hover-card

**Comment by jurrejansen** [comment]
@DonKoko @carlosvirreira any comments?

https://www.figma.com/file/ykvc0MhPeNLsrJ4tJsKosQ/Shelf-%7C-UX-Scenarios?type=design&node-id=4669-30400&mode=design

**Comment by carlosvirreira** [comment]
I very much like it - would work great on demos to show also the user type as part of our automatic logging of asset mutations on shelf
